internal/utils/trim_url.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	"net/url"
+	"strings"
+)
+
+func TrimURL(inputURL string) (string, error) {
+	parsedURL, err := url.Parse(inputURL)
+	if err != nil {
+		return "", err
+	}
+
+	hostWithoutScheme := strings.TrimPrefix(parsedURL.Host, "www.")
+	if hostWithoutScheme == "" {
```

Comment:
This works as expected - I'm getting the Host from the parsed URL and if it has www. for some reason it trims that. I may need to rename this function so it makes more sense lol but overall it will remove the scheme AND www. if the www. exists.